capstone/src/main/resources/static/script.js

```diff
@@ -416,3 +403,20 @@ function closeForm() {
 	document.getElementById("organization").style.display = "none";
 }
 
+/* function to generate divs for the calendar */
+function createCalendar() {
+  const calendar = document.getElementById("calendar");
+  for (var i = 0; i < 14; i++) {
```

Comment:
Not something we need to address in this pr but what do you think of changing the design from having for each week instead of each day? Honestly have NO idea how that'd turn out but right now having a block for each day feels a bit cluttered.

Reply:
Yeah, I actually tried formatting it into a seven column (so like by week instead of by day), and it also seems equally cluttered to be honest because then the events would be formatted vertically one after the other in a column. I'm not sure what to do.